### DISCUSSION TITLE
For processor.dig, convert machine language to assembly language?

### DISCUSSION THREAD

**Discussion by jnoche** [hneemann/Digital#741] [Q&A] [answered]
I understand that the program called Assembler can convert assembly language source code (for the Processor.dig circuit) into machine language hexadecimal source code.  But is there a simple way to convert machine language hexadecimal source code (for the Processor.dig circuit) to assembly language source code?

**Comment by hneemann** [comment, marked as answer]
You mean a disassembler?
No, I never have implemented a disassembler.

**Reply by jnoche** [reply]
Yes, that's what I meant.  Thanks for replying.
